pom.xml

```diff
@@ -1657,8 +1658,8 @@
             <id>Coverage</id>
             <dependencies>
                 <dependency>
-                    <groupId>net.sourceforge.cobertura</groupId>
-                    <artifactId>cobertura</artifactId>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>cobertura-maven-plugin</artifactId>
                     <version>${cobertura.version}</version>
```

Comment:
Everything comes up zero using 2.6 - not sure why but it seems to be a known bug and the recommendation is to go back to 2.5.2